pkg/lib/oauth/handler/handler_token.go

```diff
@@ -543,6 +566,7 @@ func (h *TokenHandler) handleAnonymousRequest(
 	}
 
 	// TODO(oauth): allow specifying scopes
+	// Note(tung): Do we allow anonymous user to use device_sso and x_app_initiated_sso_to_web?
```

Comment:
No, we do not allow device_sso in anonymous user.

And line 570 should be `["openid", "offline_access", "full-access"]`

Reply:
Let me remove the note.
I've opened https://linear.app/authgear/issue/DEV-1460 for discussion on support of device_sso (And app initiated sso)